README.md

```diff
@@ -1,143 +1,102 @@
 
-    %% source graph
-    dynamodb -->|"Retrieve all 'UNPROCESSED' DOIs"|unprocessed_dois
-    s3 -->|"Retrieve all DOIs from CSV files"| unprocessed_dois
-   
-```
+## Development
 
-### Processing of DOIs
-
-This sequence diagram depicts the processing workflow for a single DOI when the application's `deposit` command is run. 
-
-```mermaid
----
-title:
----
-
-sequenceDiagram
-    participant wiley-app as Wiley application
-    participant s3 as Amazon S3
-    participant dynamodb as Amazon DynamoDB
-    participant ses as Amazon SES
-    participant sqs as Amazon SQS
-    participant crossref as Crossref
-    participant wiley-online-lib as Wiley Online Library
-
-    wiley-app ->> dynamodb: Get DOI item from table
-    dynamodb -->> wiley-app: DOI item
-    wiley-app ->> dynamodb: Increment (+1) process_attempts in table for DOI
-    wiley-app ->> crossref: Get metadata via API request
-    crossref -->> wiley-app: Crossref metadata
-    wiley-app ->> wiley-app: Map Crossref metadata to DSpace metadata
-    wiley-app ->> wiley-online-lib: Get PDF from Wiley via API request
-    wiley-online-lib -->> wiley-app: PDF
-    wiley-app ->> s3: Upload DSpace metadata JSON and Wiley PDF
-    wiley-app ->> sqs: Send message to 'dss-wiley-output' queue
-    wiley-app ->> wiley-app: Filter log streams to ERROR messages
-    wiley-app ->> ses: Send an email to stakeholders (subject: "Automated Wiley deposit errors")
-```
+- To preview a list of available Makefile commands: `make help`
+- To install with dev dependencies: `make install`
+- To update dependencies: `make update`
+- To run unit tests: `make test`
+- To lint the repo: `make lint`
+- To run the app: `pipenv run awd --help`
 
-## `listen`
-
-### Processing of DSS messages
-This sequence diagram depicts the processing workflow for a single message from the [dspace-submission-service](https://github.com/MITLibraries/dspace-submission-service/tree/main) when the application's `listen` command is run.
-
-```mermaid
----
-title: 
----
-
-sequenceDiagram
-    participant wiley-app as Wiley application
-    participant dynamodb as Amazon DynamoDB
-    participant ses as Amazon SES
-    participant sqs as Amazon SQS
-
-    wiley-app ->> sqs: Retrieve DSS message for the DOI
-    sqs -->> wiley-app: DSS message 
-    wiley-app ->> dynamodb: Get DOI item from table
-    dynamodb -->> wiley-app: DOI item
-    wiley-app ->> wiley-app: Log result of DSpace submission for the DOI
-    alt submission was successful
-        wiley-app ->> dynamodb: Set DOI's status to 'SUCCESS'
-    else submission result in error
-        alt DOI's process_attempts >= retry threshold 
-            wiley-app ->> dynamodb: Set DOI's status to 'FAILED'
-        else DOI's process_attempts < retry threshold
-            wiley-app ->> dynamodb: Reset DOI's status to 'UNPROCESSED'
-        end
-    end
-    wiley-app ->> ses: Send an email to stakeholders (subject: "DSS results")
+## Environment Variables
 
+### Required
```

Comment:
I like the inclusion and formatting of env vars in the main README.  Not a request for this PR, but maybe a topic for maintenance weeks could be settling on a formatting approach for all repositories when it comes to env vars.  I think we slowly coalescing around this code block, but could probably further refine a bit.

Reply:
Yeah. I think coming to a decision on the format of the "Environment Variables" section is a good idea (as well as capitalization rules for section headers). Both small things but I do think consistency/uniformity helps README navigation. For this README, I followed what you'd been doing (e.g. in `geo-harvester` but added a newline between each variable).

Reply:
Yeah, hilariously, it's the newline -- again, don't think we should change it here! -- that kind of threw me.  I've come to expect env var sections (basically code) to be a little more compact and terse.  But it's ultimately just an agreed upon style, and looking forward to a style guide of sorts (even if just codified in our maintenance week updates flow).